packages/flutter/lib/src/material/checkbox.dart

```diff
@@ -609,12 +641,12 @@ class _RenderCheckbox extends RenderToggleable {
       ..strokeWidth = _kStrokeWidth;
   }
 
-  void _drawBorder(Canvas canvas, RRect outer, double t, Paint paint) {
+  void _drawBorder(Canvas canvas, Rect outer, double t, Paint paint) {
```

Comment:
This method doesn't appear to draw anything.

Reply:
This function is drawing the outer path of the checkbox.

Reply:
And all the other required changes are done :)